core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerHttpClient.java

```diff
@@ -100,6 +100,23 @@
         return newDecorator(CircuitBreakerMapping.perHostAndMethod(factory), strategy);
     }
 
+    /**
+     * Returns a new {@link CircuitBreakerHttpClientBuilder} instance with
+     * the specified {@link CircuitBreakerStrategy}.
+     */
+    public static CircuitBreakerHttpClientBuilder builder(CircuitBreakerStrategy strategy) {
+        return new CircuitBreakerHttpClientBuilder(strategy);
+    }
+
+    /**
+     * Returns a new {@link CircuitBreakerHttpClientBuilder} with
+     * the specified {@link CircuitBreakerStrategy} and {@link CircuitBreakerStrategyWithContent}.
```

Comment:
Could you remove `{@link CircuitBreakerStrategy}`?

Reply:
Of course. Can I ask you why It should be removed?(just wanna learn the reasons behind it and to prevent to write similar wrong code that I might write) 
Because It is enough clear only with parameter?